.ci_helpers/run-test.py

```diff
@@ -101,8 +100,6 @@
                 f"echopype/{module}/*",
                 f"echopype/tests/{module}/*",
             ]
-        if "extra_globs" in mod_extras:
-            file_globs = file_globs + mod_extras["extra_globs"]
```

Comment:
This if section should be okay. This is in case there are exceptions like these in the future, if not then the dict above can be changed to just a list, really.

Reply:
awesome, this is exactly the place i wasn't sure if I should delete!